src/prefect_server/api/cloud_hooks.py

```diff
@@ -76,12 +76,16 @@ async def create_cloud_hook(
         if set(config or {}) != {"url"}:
             raise ValueError("Invalid config; expected `url`")
     elif type == "TWILIO":
-        if set(config or {}) != {
-            "account_sid",
-            "auth_token",
-            "messaging_service_sid",
-            "to",
-        } or not config.get("to"):
+        if (
+            set(config or {})
+            != {
+                "account_sid",
+                "auth_token",
+                "messaging_service_sid",
+                "to",
+            }
+            or not config.get("to")
+        ):
```

Comment:
@joshmeek reformatted code like this ^^^ is why it was pinned; this looks awful

Reply:
I actually think it looks better! THE FUTURE IS NOW